src/main/webapp/app/service/firebase/firebase-gene-service.ts

```diff
@@ -485,4 +497,46 @@ export class FirebaseGeneService {
   updateObject = async (path: string, value: any) => {
     await this.firebaseRepository.update(path, value);
   };
+
+  saveAllGenes = async (isGermlineProp: boolean) => {
+    const drugLookup = (await this.firebaseRepository.get(FB_COLLECTION.DRUGS)).val() as Record<string, Drug>;
+    const geneLookup =
+      ((await this.firebaseRepository.get(isGermlineProp ? FB_COLLECTION.GERMLINE_GENES : FB_COLLECTION.GENES)).val() as Record<
```

Comment:
The logic of getting gene collection based on isGermline should be moved to firebase utils. We do something similar in firebase-review-utils.tsx.

Reply:
I see there's a `getGenePathFromValuePath` function, but it does not have logic to actually fetch.
Did you want me to use that function?

Reply:
I think this one is just to see which gene collection to use.